### DISCUSSION TITLE
[shadow server] Clipboard supported?

### DISCUSSION THREAD

**Discussion by ptosch** [FreeRDP/FreeRDP#9272] [General]
Hello,

I am not sure if this is a bug.

Unfortunately clipboard sharing does not work for me at all. Client and server are both Linux, on the server I use FreeRDP version 3.0.0-dev3 (git 5be5553e0), freerdp-shadow-cli. The client is Remmina because the Freerdp client ignores the clipboard (why?).
Both systems are Debian 6.1.38-2 (2023-07-27) x86_64 GNU/Linux (Devuan).

The following is in the debuglog when I copy something to the clipboard on the client:
[16:35:04:652] [14195:00003778] [WARN][com.freerdp.core.server] - [WTSReceiveChannelData]: unknown channelId 1006 ignored

So far  freerdp-shadow-cli replaces vnc completely. Only issue left is clipboard.
Any help is appreciated.

Complete log:
[freerdp.txt](https://github.com/FreeRDP/FreeRDP/files/12269816/freerdp.txt)

**Comment by akallabeth** [comment]
shadow does not implement the clipboard, that is why the channel is ignored.
